kqueen/engines/gce.py

```diff
@@ -143,26 +143,38 @@ def __init__(self, cluster, **kwargs):
         self.project = self.service_account_info.get('project_id', '')
         self.zone = kwargs.get('zone', '-')
         self.cluster_id = 'a' + self.cluster.id.replace('-', '')
+
+        # Generate metadata for Network Policies if empty
+        if not isinstance(cluster.metadata.get('network_policy', None), dict):
+            self.cluster.metadata['network_policy'] = network_meta = {}
+            network_meta['provider'] = kwargs.get('network_policy', 'PROVIDER_UNSPECIFIED')
+            network_meta['enabled'] = True if network_meta['provider'] != 'PROVIDER_UNSPECIFIED' \
+                else False
+            logger.debug('Generate metadata for network policies:{}'
+                         .format(cluster.metadata['network_policy']))
+            self.cluster.save()
+
+        meta = self.cluster.metadata
```

Comment:
Looks like you use it only to get network policy, so you can declare `self.cluster.metadata['network_policy']`

Reply:
in future i prefer to use that for another cases